base/ltsockets.dtx

```diff
@@ -645,6 +645,43 @@
 %   \cs{LogSocket} and \cs{socket_log:n} only differ in that they don't stop.
 % \end{function}
 %
+%
+% It is sometimes necessary/helpful to know if a particular socket or
+% plug exists (or is assigned to a certain socket) and based on that
+% take different actions.
+%
+% \begin{function}[EXP]{\IfSocketExistsTF,\socket_if_exist:nTF}
```

Comment:
I'm not sure how well this works for the non-L3 names, but maybe use the `TF` key of the `function` environment or the `pTF` (since the `p`-variants are also defined). If `pTF` isn't usable because of the 2e names how should this be documented? (also applicable to the other new conditionals.

Reply:
It doesn't which is why I have chosen the easy way out. Eventually this should be fixed in the l3doc class, but this has low priority for me.